Source/UI/NewUI/CommonUI/View/TableViewCell/CKOCellTextField.swift

```diff
@@ -1,65 +1,51 @@
 import UIKit
-import PhoneNumberKit
 
-protocol BillingFormTextFieldCellDelegate: AnyObject {
+protocol CKOCellTextFieldDelegate: AnyObject {
     func updateCountryCode(code: Int)
     func textFieldShouldBeginEditing(textField: UITextField)
     func textFieldShouldReturn()
     func textFieldShouldEndEditing(textField: UITextField, replacementString: String)
     func textFieldShouldChangeCharactersIn(textField: UITextField, replacementString string: String)
 }
 
-final class CKOTextFieldCell: UITableViewCell {
-    weak var delegate: BillingFormTextFieldCellDelegate?
+final class CKOCellTextField: UITableViewCell {
+    weak var delegate: CKOCellTextFieldDelegate?
     var cellStyle: BillingFormCell? = nil
-    var style: CKOTextFieldCellStyle? = nil
+    var style: CKOCellTextFieldStyle? = nil
 
-    
-    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
-        super.init(style: style, reuseIdentifier: reuseIdentifier)
-    }
-    
-    func update(cellStyle:BillingFormCell, style: CKOTextFieldCellStyle, tag: Int) {
-        self.cellStyle = cellStyle
-        self.style = style
-        self.tag = tag
-        setupViewsInOrder()
+    private var mainView: UIView
+    init(mainView: UIView) {
+        self.mainView = mainView
+        super.init(style: .default, reuseIdentifier: nil)
+        self.setupViewsInOrder()
     }
-    
 
     @available(*, unavailable)
     required init?(coder _: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
-    private lazy var paymentInputView: UIView = {
-        guard let cellStyle = cellStyle, let style = style else { return UIView() }
-        let view = CKOTextFieldView(type: cellStyle, tag: tag, style: style ,delegate: self)
-        view.translatesAutoresizingMaskIntoConstraints = false
-        return view
-    }()
-    
 }
 
-extension CKOTextFieldCell {
+extension CKOCellTextField {
     
     private func setupViewsInOrder() {
-        contentView.addSubview(paymentInputView)
+        mainView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(mainView)
```

Comment:
should probably not exist with correct dequeuing behaviour

Reply:
Please read this 
 https://www.codementor.io/@warrenburton/do-we-need-to-dequeue-tableview-cells-w8szp55uq

Reply:
Please read the conclusion from that:
> If frame rate (smoothest scroll performance) is important to you then you should use cell dequeueing.

Reply:
Updated